lib/kernel/src/inet.erl

```diff
@@ -144,10 +144,13 @@
 -type inet_backend() :: {'inet_backend', 'inet' | 'socket'}.
 
 -type socket_setopt() ::
-        gen_sctp:option() | gen_tcp:option() | gen_udp:option().
+        gen_sctp:setoption() | gen_tcp:option() | gen_udp:option().
+
+-type socket_optval() ::
```

Comment:
This type needs to be inlined into the `getopts/2` documentation or documented at the top like `socket_setopt`.

Reply:
I expanded everything out.
It is now consistent with other documentation